### DISCUSSION TITLE
Site's been screwy for hours

### DISCUSSION THREAD

**Discussion by Klokinator** [civitai/civitai#164] [General]
Are you guys getting DDOS'd? Connection issues, other problems. I can't download anything either. Keep getting XML error pages or other wrong links.

**Comment by JustMaier** [comment]
We went down for a few hours on Friday. Here's what we posted about it in our Discord that day.

---

> So, regarding the downtime today - We were attacked by an unknown class of AI. I wish it was that cool. Actually, earlier today our hosting service had a short period of sporadic issues for the duration of about 2 hours. The periodic issues caused our authentication system (Next Auth) to struggle communicating with itself. As it tried repeatedly to connect as users hit the site, it eventually lead the system to die and reboot after which it immediately ran into the same issues since the error responses from the previous instance of the site were still being fed to the auth system. Basically it was a spiral of death (insert AI generated image here). Eventually after stopping things for a period of time by turning on "maintenance mode" when we brought the site back the auth system correctly responded. Things were slow as the system warmed back up, but then things seemed to be running smoothly. We've found some things we can adjust with the auth system and we're going to be reviewing our hosting infrastructure as well.

**Comment by Klokinator** [comment]
Late reply on my end but thanks for the response!
